src/components/HorizontalStepper/HorizontalStepper.stories.tsx

```diff
@@ -87,13 +86,17 @@ export const Interactive: StoryObj<Args> = {
 export const HorizontalSteps: StoryObj<Args> = {
   render: () => (
     <ol className="p-0">
-      <HorizontalStep
+      <HorizontalStepper.Step
```

Comment:
Not thrilled with the component name here. `Horizontal` is actually a variant embedded into the name. There could be a vertical one at some point, and switching to it could be a responsive behavior even on the horizontal thing. In fact, `TimelineNav` and this thing have a few items in common...

Renaming for now, but we should revisit this component with design at some point